server/routes/induction/create/previousWorkExperienceTypesCreateController.ts

```diff
@@ -69,29 +69,14 @@ export default class PreviousWorkExperienceTypesCreateController extends Previou
 
 /**
  * Builds and returns a Page Flow Queue to show the Details page for each work experience type. The list of pages to be
- * added to the queue is derived by the difference between the work types on the updated induction (ie. after the
```

Comment:
This was the problem area, in that it was trying to be too smart for it's own good!
What it tried to do was work out the diff between the previous work experience types on the induction between when the page was first loaded vs when it was submitted.

This worked great in a linear create journey, where you start off with no work experiences ticked, you set some, and submit. Therefore all the ones you ticked are the resultant diff

And it worked great when clicking Change from Check Your Answers, in that it would work out the new ones you'd selected and only show the detail page for those newly added work experience types.

Where it doesn't work, and the cause of this bug, is when there are no differences between when the page was loaded and the page being submitted. 
EG. if you click Change from Check Your Answers, but dont actually change anything
or (in this case), you click Back from the work experience detail page (ie. the next page after this one), then click Continue to move forward again without actually making any changes.

The simplest option here was to remove the complexity of it trying to be smart. We no longer try to work out the diff, and simply show the work experience detail page for each type submitted on the form.